specification/web/resource-manager/Microsoft.Web/2016-03-01/ResourceProvider.json

```diff
@@ -254,6 +254,41 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/listSitesAssignedToHostName": {
+      "post": {
+        "summary": "List all apps that are assigned to a hostname.",
+        "description": "List all apps that are assigned to a hostname.",
+        "operationId": "ListSitesAssignedToHostName",
```

Comment:
Incorrect operationId pattern, ideally this should be
`List_Identifiers` or maybe `List_IdentifiersAssignedToHostName`